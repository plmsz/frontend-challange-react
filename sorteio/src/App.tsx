import { DrawContextProvider } from './context/DrawContext'
import GlobalStyle from './styles/GlobalStyle'
import { Routes } from './routes'

function App() {
  return (
    <>
      <DrawContextProvider>
        <GlobalStyle />
        <Routes />
      </DrawContextProvider>
    </>
  )
}

export default App
