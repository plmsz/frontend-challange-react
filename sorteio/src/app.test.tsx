import { render, screen } from './tests/renderWithBrowserRouter'
import App from './App'

it('renders home correctly', async () => {
  const { asFragment } = render(<App />, '/')

  const title = await screen.findByRole('heading', { name: /mega-sena/i })
  expect(title).toBeInTheDocument()
  expect(asFragment()).toMatchSnapshot('create')
})
