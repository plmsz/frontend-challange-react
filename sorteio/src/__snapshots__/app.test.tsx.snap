// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders home correctly: create 1`] = `
<DocumentFragment>
  <div
    class="sc-papXJ gkSmcJ"
  >
    <header
      class="sc-gKXOVf kkZcHQ"
    >
      <div
        class="sc-gsnTZi dkJDGY"
      >
        <select
          aria-label="Loterias"
          class="sc-jSMfEi RuIrY"
          name="lotteryDropdown"
        >
          <option
            value="mega-sena"
          >
            mega-sena
          </option>
          <option
            value="quina"
          >
            quina
          </option>
          <option
            value="lotofácil"
          >
            lotofácil
          </option>
          <option
            value="lotomania"
          >
            lotomania
          </option>
          <option
            value="timemania"
          >
            timemania
          </option>
          <option
            value="dia de sorte"
          >
            dia de sorte
          </option>
        </select>
        <div
          class="sc-dkzDqf lfMgJK"
        >
          <picture>
            <source
              media="(max-width: 768px)"
              srcset="logoPrimary.svg"
            />
            <source
              media="(min-width: 768px)"
              srcset="logoSecondary.svg"
            />
            <img
              alt="logo - trevo de 4 folhas"
              src="logoPrimary.svg"
            />
          </picture>
          <h1>
            mega-sena
          </h1>
        </div>
        <h2
          class="titleMobile"
        >
          CONCURSO Nº 1000
        </h2>
        <div
          class="titleDesktop"
        >
          <h2>
            CONCURSO
          </h2>
          <p>
            1000 - 
          </p>
        </div>
      </div>
    </header>
    <main
      class="sc-ftvSup ccABDJ"
    >
      <h1
        class="sc-hKMtZM cQIvia"
        color="quina"
      >
        Aposte, hoje pode ser o seu dia!
      </h1>
    </main>
  </div>
</DocumentFragment>
`;
